com.unity.mobile.notifications/Runtime/iOS/QueryLastRespondedNotificationOp.cs

```diff
@@ -0,0 +1,135 @@
+using System;
+using UnityEngine;
+
+namespace Unity.Notifications.iOS
+{
+    /// <summary>
+    /// The state of the query for last responded notification.
+    /// Returned by <see cref="QueryLastRespondedNotificationOp.State"/>
+    /// </summary>
+    public enum QueryLastRespondedNotificationState
```

Comment:
Is this intentional that exactly the same enum is in com.unity.mobile.notifications/Runtime/Unified/QueryLastRespondedNotificationOp.cs ?  and both are even public.

Reply:
One is iOS specific API, other works on both Android and iOS. The later is in assembly that depends on former. Same for the other.
Unified APIs wrap the common parts of Android and iOS APIs (leaves specific things out, but if you only need basic stuff, it makes writing script much much simpler).

Reply:
Shouldn't then iOS part derive from unified part (Unity.Notifications.iOS.QueryLastRespondedNotificationOp derive from  Unity.Notifications.QueryLastRespondedNotificationOp)  ? and expand stuff where needed. 

Common sense say, we shouldn't duplicate code if possible

Reply:
It can't do that now because the dependency is the opposite.

Reply:
Maybe something to refactor for the future? maybe at least I would be possible to not duplicate the enum?

Reply:
Yes, we probably could have one more assembly for common stuff. But that requires structural changes. Right now I'd like to not do them given that we will likely have to do them soon anyway, so one break rather than two.

Reply:
Maybe worth creating JIRA items for next major version bump?